talentsearch/resources/js/stories/RequestForm.stories.tsx

```diff
@@ -1,21 +1,106 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
-import { fakeDepartments, fakePoolCandidateFilters } from "@common/fakeData";
+import { fakeDepartments } from "@common/fakeData";
 import { RequestForm } from "../components/request/CreateRequest";
 import {
   CreatePoolCandidateSearchRequestInput,
   PoolCandidateFilter,
+  WorkRegion,
 } from "../api/generated";
 
 const stories = storiesOf("Search Request Form", module);
 
 stories.add("Search Request Form", () => {
+  const poolCandidateFilter: PoolCandidateFilter = {
```

Comment:
Lets try to create new stories with the `export { } as Meta` style. See SearchPage.stories.tsx. Defining stories like that makes the inputs automatically editable, and makes it easier to create new versions of stories later.